.github/workflows/lint.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v1
       with:
-        python-version: 3.6
+        python-version: 3.7
```

Comment:
I'd just run on the latest Python, saves updating every year for EOL dates
```suggestion
        python-version: 3
```

Reply:
I used `>=3.7` just in case there exist runners with 3.6 would be picked by default.